images/jupyter-singleuser/README.md

```diff
@@ -3,9 +3,21 @@
 This is the notebook image that individual users are served
 via JupyterHub.
 
-## Building and pushing manually
+## Testing Changes
+
+A person with Docker set up locally can build a new version of the image at any time after making changes.
```

Comment:
Same comments from the dask one about version number

Reply:
Resolved in [696ac34](https://github.com/cal-itp/data-infra/pull/3156/commits/696ac3432bae8a47626ed6bf35dc191c0254d2b9)